python/ray/_private/services.py

```diff
@@ -290,6 +280,123 @@ def find_redis_address_or_die():
     return redis_addresses.pop()
 
 
+def find_gcs_address():
+    """
+    Attempts to find all valid Ray GCS address on this node.
+
+    Returns:
+        Set of detected Redis instances.
+    """
+    # Currently, this extracts the --gcs_address flag from the command
+    # that launched the raylet running on this node, if any. Anyone looking to
+    # edit this function should be warned that these commands look like, for
+    # example:
+    # /usr/local/lib/python3.8/dist-packages/ray/core/src/ray/raylet/raylet
+    # --gcs_address=123.456.78.910:6379 --node_ip_address=123.456.78.910
```

Comment:
cc @iycheng Assuming `--gcs_address` format contains ip:port, which is different from `--redis_address` on raylet that only contains ip.